shuffle/cli/src/account.rs

```diff
@@ -143,7 +143,13 @@ pub async fn get_treasury_account(
     client: &DevApiClient,
     root_key_path: &Path,
 ) -> Result<LocalAccount> {
-    let treasury_account_key = load_key(root_key_path);
+    let treasury_account_key = if root_key_path.exists() {
+        load_key(root_key_path)
+    } else {
+        return Err(anyhow!(
+            "A node hasn't been created yet! Run shuffle node first"
```

Comment:
nit: This could be improved by using guards, as described here: https://betterprogramming.pub/refactoring-guard-clauses-2ceeaa1a9da

```rust
if !root_key_path.exists() {
     return Err(anyhow!(
            "A node hasn't been created yet! Run shuffle node first"
            );
   }
   let treasury_account_key = load_key(root_key_path);
```